File: Kth Largest Element In An Array/Kth Largest Element In An Array.java
class Solution {
    public int findKthLargest(int[] nums, int k) {
        PriorityQueue<Integer> pq = new PriorityQueue<>();
        
        for(int i = 0; i < k; i++) {
            pq.add(nums[i]);
        }
        
        for(int i = k; i < nums.length; i++) {
            if (nums[i] >= pq.peek()) {
                pq.add(nums[i]);
                pq.remove(pq.peek());
            }
        }
        
        return pq.peek();
    }
}

File: Kth Largest Element In An Array/Kth Largest Element In An Array.py
class Solution:
    def findKthLargest(self, nums: List[int], k: int) -> int:
        arr = nums[0:k]
        heapq.heapify(arr)
        
        for i in range(k, len(nums)):
            if (nums[i] >= arr[0]):
                heapq.heappush(arr, nums[i])
                heapq.heappop(arr)
        
        return arr[0]